Traducir.Web/src/App/Components/SuggestionsHistory.tsx

```diff
@@ -57,9 +57,9 @@ export default class SuggestionsHistory extends React.Component<ISuggestionsHist
 
     public async userChanged(location: Location): Promise<void> {
         const userId = location.pathname.split("/").pop();
-        const stateId = location.search.substring(location.search.lastIndexOf("=") + 1);
+        const state = location.search.substring(location.search.lastIndexOf("=") + 1);
```

Comment:
ouch, I know we talked about it... but I totally forgot that I was already using `parse` from `query-string` [here](https://github.com/g3rv4/Traducir/blob/master/Traducir.Web/src/App/Components/Filters.tsx#L243). I like that better because it's less brittle if we later add more parameters (or if say an extension added a cache breaking parameter).

Reply:
thanks god I didn't added another parse library (I didn't find this one)... I have to make a model for this to work. That's ok with you?

Reply:
you should be good with just declaring the interface on this file (it's not like it's going to be used anywhere else), right? or am I missing something?

Reply:
Sure.. just checking! ;)